src/frontend/Komponenter/Behandling/Inngangsvilkår/Aleneomsorg/AleneomsorgInfo.tsx

```diff
@@ -124,7 +126,7 @@ const AleneomsorgInfo: FC<{
                             )}
                         </Normaltekst>
                     </>
-                }
+                )}
```

Comment:
Gir det mening å legge denne OG den under i samme stønadstype-sjekk?

Reply:
Den er allerde sjekke for - et hakk opp i komponenthierarkiet: 

useEffect(() => {
        if (stønadstype === Stønadstype.BARNETILSYN) {
            axiosRequest<IBarnMedLøpendeStønad, null>({
                method: 'GET',
                url: `/familie-ef-sak/api/tilkjentytelse/barn/${behandlingId}`,
            }).then((respons: Ressurs<IBarnMedLøpendeStønad>) => {
                settBarnMedLøpendeStønad(respons);
            });
        }
    }, [axiosRequest, behandlingId, stønadstype, settBarnMedLøpendeStønad]);

Reply:
sant det :)